### PR TITLE
Added operation to insert database instance

### PR DESCRIPTION
**Pull request by ArthyVenk**
Also added a python operation to frame the insert instance body

### CODE REVIEW

**Review by KalyankarRajesh — COMMENTED** [review]
change the namespace of insert_database_instance operation to io.cloudslang.google.databases.instances

**Review by KalyankarRajesh — COMMENTED** [review]
change the namespace for insert_instance_request_body to instances and update in insert_instance operation

**Review by KalyankarRajesh — APPROVED** [review]

File: content/io/cloudslang/google/databases/instances/insert_database_instance.sl [hunk 1]
```diff
@@ -0,0 +1,301 @@
...
+#!                        you trust to identify other parties.  If the protocol (specified by the 'url') is not
+#!                        'https' or if trust_all_roots is 'true' this input is ignored.
+#!                        Default value: '..JAVA_HOME/java/lib/security/cacerts'
+#!                        Format: Java KeyStore (JKS)
+#!                        Optional
+#! @input trust_password: The password associated with the trust_keystore file. If trust_all_roots is false
+#!                        and trust_keystore is empty, trust_password default will be supplied.
+#!                        Optional
+#!
+#! @output instance_json: A JSON list containing the Instance information.
```
Comment (patmante) [inline content/io/cloudslang/google/databases/instances/insert_database_instance.sl]: update the name to "database_instance_json"

**Review by patmante — CHANGES_REQUESTED** [review]

File: content/io/cloudslang/google/databases/instances/insert_database_instance.sl (hunk 1 above)
Comment (ArthyVenk) [reply content/io/cloudslang/google/databases/instances/insert_database_instance.sl]: Updated

**Review by patmante — APPROVED** [review]

**Pull request merged**